internal/app/machined/pkg/runtime/v1alpha1/board/rpi_4/config.txt

```diff
@@ -9,6 +9,8 @@ hdmi_safe:1=1
 kernel=u-boot.bin
 # Forces the kernel loading system to assume a 64-bit kernel.
 arm_64bit=1
+# Run as fast as firmware / board allows
```

Comment:
```suggestion
# Run as fast as firmware / board allows.
```

Reply:
> My only concern is that this option is not documented on https://www.raspberrypi.com/documentation/computers/configuration.html
> 
> (and while you are on it, may you please update the link on top of the file?)

Will do, its still in a pull request here: https://github.com/raspberrypi/documentation/pull/2281/files